source/synthDrivers/espeak.py

```diff
@@ -25,6 +25,15 @@
 	PhonemeCommand,
 )
 
+
+# A mapping of commonly used language codes to e-speak languages.
+# These are used when e-speak doesn't support a given language code
+# but a default alias is appropriate
+_defaultLangToLocaleMappings = {
+	"en": "en-us",

```

Comment:
eSpeak's original language is `en-gb`: it's the default choice on Linux, NVDA itself, and most users I've experienced. In my opinion, `en-us` is unsuitable for a default English dialect!

```suggestion
	"en": "en-gb",

```

Reply:
Thank you - I was unaware of this